core/server/models/session.js

```diff
@@ -0,0 +1,81 @@
+const ghostBookshelf = require('./base');
+
+const Session = ghostBookshelf.Model.extend({
+    tableName: 'sessions',
+
+    parse: function parse(attrs) {
```

Comment:
There is some inconsistency in this model

> parse: function parse
> permittedOptions()

Reply:
Good point, I prefer `name() {` and you?

Reply:
Yeah 👍